flags.yml

```diff
@@ -83,6 +83,13 @@
   contact: Query Team
   lifetime: temporary
 
+- name: Experimental Query Tracing
```

Comment:
With this being a permanent flag, I think we can just call this query tracing, leaving out "experimental"? What do you think?

Reply:
Yes, I agree, I'll change it.